accepted/00102-git-integration.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: Git integration
+- Start Date: 2024-04-23
+
+# Summary
+[summary]: #summary
+
+This is a feature which will add a git integration to Uyuni without the need to use gitfs. It would be possible to sync formulas and playbooks and use them with an existent infrastructure.
+
+# Motivation
+[motivation]: #motivation
+
+- Why are we doing this?
+The git integration is not existent or it the depends from gitfs which is unstable
+
+- What use cases does it support?
+For all users which have a lot of self-written salt formulas and ansible-playbooks.
+
+- What is the expected outcome?
+Use salt formulas and ansible playbooks which are stored on git repositories
+Use them against clients managed by Uyuni
+
+
+# Detailed design
+[design]: #detailed-design
+
+It should be designed on four parts:
+
+The first part would be done by a container that is responsible to sync the git repositories and sort it out per type of the provisioning tool.
+ - If that is a ansible playbook it would be scanned(find $path for a role infrastructure)
+ - If that is a salt formula it should be done by the states delivered by the repository
+ - Path structure:
+   - Ansible: $local/ansible/roles, $local/ansible/inventory, $local/ansible/config
+   - Salt: $local/salt/states, $local/ansible/config
+- All the data would be saved in a volume which would be shared with the suma container - e.g. $provisioning
+
+The second part it would be done for a md5/sha256sum check. If possible, also a scan that could be done for a container with e.g. clamav to check if there any viroses / trojans being imported into the clients being managed by Uyuni.
```

Comment:
The checksum part sounds like a good candidate to add (optional) ansible-sign support: https://ansible.readthedocs.io/projects/sign/en/latest/.
Not sure about the clamav part though. This seems uncommon when synching repo content.